cypress/helpers/modal/modal.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ *
+ * @param testId Optional test id to target the desired modal
+ */
+export function get(testId?: string) {
```

Comment:
Can we combine this file with `cypress/helpers/modal/index.ts`? It seems unnecessary to have two files for these.

Reply:
We could move this file to `cypress/helpers/modal.ts`. I don't remember why I created a folder for this

Reply:
I thought of a downside to the composable helper function vs Page Objects (or component class helpers). There is no strict type safety with this approach. At my last org, we used this style and didn't really run into issues, but I never experimented with if I should add a stricter type output.

For example, the return signature is `Cypress.Chainable<JQuery>` and the input of `getTitle` and `getCloseButton` are `$modal: JQuery`. Technically that could be any element. Should we be more strict about the type signature?

```ts
export interface Modal extends JQuery {
  __$el: 'Modal';
}

export function get(testId?: string): Cypress.Chainable<Modal> {
  const selector = testId ? `[data-testid='${testId}']` : `[role=dialog]`;

  return cy.get(selector) as Cypress.Chainable<Modal>;
}

export function getTitle($modal: JQuery): JQuery<HTMLHeadingElement> {
  return $modal.find('h3').first() as JQuery<HTMLHeadingElement>;
}

export function getCloseButton($modal: JQuery): JQuery<HTMLButtonElement> {
  return $modal.find('[aria-label=Close]') as JQuery<HTMLButtonElement>;
}
```

The typing is more strict, but the type error can be confusing and I don't think it adds much value.

Reply:
> We could move this file to `cypress/helpers/modal.ts`. I don't remember why I created a folder for this

I'm guessing better import ergonomics when you have an index file? Or at least that's what I would do...